src/sync/outgoingSyncUtils.js

```diff
@@ -222,6 +223,17 @@ const generateSyncData = (settings, recordType, record) => {
         type: TRANSACTION_BATCH_TYPES.translate(transaction.type, INTERNAL_TO_EXTERNAL),
       };
     }
+    case 'Message': {
+      return {
+        ID: record.id,
+        fromStoreID: settings.get(THIS_STORE_ID),
```

Comment:
does not having toStoreID work ? or does it have to be 0 ?

Reply:
ya 

<img width="1012" alt="image" src="https://user-images.githubusercontent.com/35858975/71950972-234f9280-323e-11ea-9fc6-a145efa04794.png">